### PR TITLE
Updating README.md and INSTALL.md

### PR DESCRIPTION
**Pull request by ai-fast-track**
The installation procedure has completely changed after the introduction of the new setup process. Therefore, the installation guide had to be re-written.

I updated the following files: README.md, INSTALL.md, and CONTRIBUTING.md. These are the changes:

1 - INSTALL.md: I re-wrote all the pip install section: Introduced the 3 ways of install the mantisshrimp + link the blog post

2 - README.md: 
- I added the Carbon screenshot 
- I added a link to the source code. I added an Installation Section (with a hidden tag to hide it in the Docs).
- I update the `autogen.py` to hide the installation section from the Docs because there is already an Installation entry in the navigation bar. 
- I removed the slack invitation. 
- I added a note: "We need your help" to ask people to star the project and share it

3 - I moved the Features Requests section from the README.md file to the CONTRIBUTING.md file

### CODE REVIEW

**Comment by oke-aditya** [comment]
Are changes from #233 taken into account ?

**Comment by ai-fast-track** [comment]
No because the installation info is out dated. I commented on JCustin PR:  https://github.com/lgvaz/mantisshrimp/pull/233

File: INSTALL.md [hunk 1]
```diff
@@ -1,30 +1,58 @@
...
+pip install mantisshrimp[all]
 ```
 
-Installing albumentations package
+#### Mantisshrimp + Fastai
+To install mantisshrimp package and only the Fastai library, run the following command:
 
 ```bash
-pip install albumentations
+pip install mantisshrimp[fastai]
```
Comment (lgvaz) [inline INSTALL.md:22]: Should we add a comment that this is only going to be available in a few days?

Maybe point to git install for now

Comment (lgvaz) [reply INSTALL.md:22]: Or to local install, since I think most users right now are also contributors

Comment (oke-aditya) [reply INSTALL.md:22]: Change it to
git clone  --depth=1 mantisshrimp
the repo. pip install .[all] for now.

Depth 1 will limit the .git thingie for end user.

Comment (ai-fast-track) [reply INSTALL.md:22]: Thank you @oke-aditya for reminding me. Yesterday, I read tweets from Jeremy and Hamel (Github) suggesting that, and then I forgot to added it.

Comment (lgvaz) [reply INSTALL.md:22]: Ow, the git thingy is 200mb, I don't think it's a problem for development.

**Comment by oke-aditya** [comment]
Just check once if git clone with depth 1 is working. Then we are ready to merge

**Pull request merged**